app/views/application_submissions/show.haml

```diff
@@ -1,33 +1,38 @@
 %h1 #{@record.position.name} Application Record
 
-submitted
-= format_date_time @record.created_at
-by:
-= @record.user.name_and_email
+%p submitted #{format_date_time @record.created_at}
+%p by #{@record.user.name_and_email}
 = button_to 'Print this page',
-  application_submission_path(@record, format: :pdf), method: :get
+  application_submission_path(@record, format: :pdf),
+  method: :get,
+  class: 'btn btn-secondary'
 - if @record.rejected?
-  = button_to 'Re-review Applicant',
-    unreject_application_submission_path(@application_submission, id: @record.id)
+  %p
+    = button_to 'Re-review Applicant',
+      unreject_application_submission_path(@application_submission, id: @record.id),
+      class: 'btn btn-primary'
 - if !@record.saved_for_later? && @record.note_for_later.present?
-  %em
-    This application had previously been saved for later review. Your note:
-  = @record.note_for_later
-.data_container
-  .data_row
-    %h2.prompt_item Question
-    %h2.response_item Response
-  - @record.data.each do |prompt, response, data_type, question_id|
-    - if data_type == 'heading'
-      .data_row
-        .heading_item= prompt.upcase
-    - elsif data_type == 'explanation'
-      .data_row
-        .explanation_item= render_markdown prompt
-    - else
-      .data_row
-        .prompt_item= prompt
-        .response_item= response
+  %p
+    %i This application had previously been saved for later review. Your note:
+  %p
+    = @record.note_for_later
+%table.table.table-bordered.application-record-table
+  %thead
+    %tr
+      %th Questions
+      %th Responses
+  %tbody
+  - @record.data.each do |prompt, response, data_type, _question_id|
+    %tr
+      - if data_type == 'heading'
+        %td
+          %b=prompt.upcase
+      - elsif data_type == 'explanation'
+        %td
+          %i= render_markdown prompt
+      - else
+        %td=prompt
+      %td=response
```

Comment:
One last thing, before this PR, headings and explanations spanned both columns:

```haml
%tr
  - if data_type == 'heading'
    %td{colspan=2}
      %b= prompt.upcase
  -elsif data_type == 'explanation'
    %td{colspan=2}
      %i= render_markdown prompt
  -else
    %td= prompt
    %ed= response
```

Reply:
you mean `%td{colspan: 2}` but yes